src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangle.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.6);
+$mainColor: $white;
+$boxShadow: 0 0 20px 0 rgba($black, 0.3);
+$borderSize: 2px;
+
+.rectangle {
+    position: absolute;
+    background: transparent;
+    border: $borderSize solid $mainColor;
+    z-index: 1000;
+    cursor: move;
+    box-sizing: content-box;
```

Comment:
Is that necessary? I think it would be nice to not be inconsistent with the box-modell we are using...